src/continuum_absorption/absorption_ff_metal.jl

```diff
@@ -0,0 +1,458 @@
+                  -0.066 -0.035  0.018  0.067  0.101  0.183;
+                  -0.066 -0.037  0.012  0.058  0.091  0.173;
+                  -0.066 -0.040  0.006  0.049  0.082  0.164;
+                  -0.066 -0.042  0.001  0.042  0.074  0.157;
+                  -0.066 -0.044 -0.004  0.036  0.067  0.150;
+                  -0.065 -0.045 -0.007  0.030  0.061  0.144;
+                  -0.064 -0.046 -0.011  0.025  0.056  0.139;
+                  -0.063 -0.047 -0.014  0.020  0.051  0.135;
+                  -0.062 -0.048 -0.016  0.017  0.048  0.131;
+                  -0.061 -0.048 -0.018  0.014  0.045  0.128;
+                  -0.059 -0.047 -0.019  0.011  0.042  0.126;
+                  -0.057 -0.047 -0.020  0.009  0.040  0.124;
+                  -0.055 -0.046 -0.020  0.008  0.039  0.123;
+                  -0.053 -0.045 -0.021  0.007  0.038  0.123;
+                  -0.051 -0.044 -0.020  0.006  0.038  0.123;
+                  -0.048 -0.042 -0.020  0.006  0.038  0.123;
+                  -0.045 -0.040 -0.019  0.006  0.039  0.124;
+                  -0.039 -0.035 -0.016  0.008  0.042  0.128;
+                  -0.032 -0.030 -0.012  0.011  0.046  0.133]
+
+    LinearInterpolation((T_vals, σ_vals), table_vals, extrapolation_bc=Throw())
+end
+
+_Mg_I_ff(ν::Real, T::Real, ndens_MgII::Real, nₑ::Real) =
+    metal_ff_absorption_departure(ν, T, 1, ndens_MgII, nₑ, _departure_term_Mg_I_ff)
+
+
+"""
+    metal_ff_absorption(ν, T, Z, ni, ne)
+
+Computes the free-free linear absorption coefficient (in cm⁻¹) of a metal species for which a 
+departure term is specified.
+
+A free-free interaction is named as though the species interacting with the free electron had one 
+more bound electron (in other words it's named as though the free-electron and ion were bound 
+together). The `Z` argument and `ni` values should be specified for the species that actually
+participates in the reaction. For completeness 2 examples provided below:
+- Si I ff absorption: `ni` holds the number density of Si II, and `Z=1` (net charge of Si II)
+- Si II ff absorption: `ni` holds the number density of Si III, and `Z=2` (net charge of Si III)
+
+This function simply wraps the hydrogenic free-free calculation.
+
+# Arguments
+- `ν`: frequency in Hz
+- `T`: temperature in K
+- `Z::Integer`: the net charge of the ion species (that participates in the interaction).
+- `ni`: the number density of the ion species (that participates in the interaction) in cm⁻³.
+- `ne`: the number density of free electrons.
+"""
+metal_ff_absorption(ν, T, Z, ni, ne) = hydrogenic_ff_absorption(ν, T, Z, ni, ne)
+
+
+
+"""
+    all_metal_ff_absorption(ν::Real, T::Real, number_densities::Dict, ne::Real,
+                            exclude_species_with_departure_terms::Bool = false)
+
+Computes the free-free linear absorption coefficient (in cm⁻¹) of metal species. This only includes
+free-free absorption for species with a net positive charge of 1, 2, or 3. It also skips Hydrogenic
+atoms.
```

Comment:
I don't think He is considered a metal.  I'm ambivalent about whether it should be included here, but if it is it should probably be mentioned explicitly.